app-modules/task/src/Filament/Resources/TaskResource/Pages/EditTask.php

```diff
@@ -57,8 +58,9 @@ public function form(Form $form): Form
     protected function getHeaderActions(): array
     {
         return [
-            Actions\ViewAction::make(),
-            Actions\DeleteAction::make(),
+            TaskViewHeaderAction::make('view')
+                ->modalSubmitAction(false),
```

Comment:
I think this can be moved to chain off of `infolist` in `setup`

Reply:
yes done.